src/courseware/course/Course.jsx

```diff
@@ -101,16 +103,22 @@ function Course({
       </section>
 
       <AlertList topic="sequence" />
-      <Sequence
-        unitId={unitId}
-        sequenceId={sequenceId}
-        courseId={courseId}
-        unitNavigationHandler={unitNavigationHandler}
-        nextSequenceHandler={nextSequenceHandler}
-        previousSequenceHandler={previousSequenceHandler}
-        //* * [MM-P2P] Experiment */
-        mmp2p={MMP2P}
-      />
+      <section className="sequence-container d-inline-flex flex-row">
```

Comment:
Did you move the Sidebar component? @anfbermudezme

Reply:
I directly add `<OutlineSidebar />` to maintain the state of the sidebar, since adding it in sequence when navigating between subsections would also change the state of the sidebar.